codeGenerator/src/FRCrobotCodeGen302/MainForm.cs

```diff
@@ -39,11 +39,26 @@ public MainForm()
             valueTextBox.Location = valueNumericUpDown.Location;
 
             this.Text += " Version " + ProductVersion;
+
+            //try to load cached configuration.xml
+            addProgress("Trying to load cached configuration.xml");
+            if (File.Exists(Path.GetTempPath() + "DragonsCodeGeneratorCache.txt"))
```

Comment:
So as not to repeat Path.GetTempPath() + "DragonsCodeGeneratorCache.txt"
do:
string cacheFile = Path.GetTempPath() + "DragonsCodeGeneratorCache.txt";
and then use cacheFile in lines 45 and 47